src/Controls/ScrollableSegmentedControl.cs

```diff
@@ -56,6 +55,16 @@ public ScrollableSegmentedControl(IEnumerable<UIImage> sectionImages, IEnumerabl
 
         public Func<ScrollableSegmentedControl, string, int, bool, NSAttributedString> TitleFormatter { get; set; }
 
+
+        public string SelectedTitle
+        {
+            get
+            {
+                if (sectionTitles.Count > selectedIndex)
+                    return sectionTitles[selectedIndex];
```

Comment:
Build fails as `selectedIndex` is missing

Reply:
Weird! It compiled on my machine, I have submitted a change for this.